tools/shared.py

```diff
@@ -1035,11 +1035,14 @@ def get_llvm_target():
 #   -Wno-implicit-function-declaration
 COMPILER_OPTS += ['-Werror=implicit-function-declaration']
 
-USE_EMSDK = not os.environ.get('EMMAKEN_NO_SDK')
 
-if USE_EMSDK:
-  # Disable system C and C++ include directories, and add our own (using -idirafter so they are last, like system dirs, which
-  # allows projects to override them)
+def emsdk_opts():
+  if os.environ.get('EMMAKEN_NO_SDK'):
+    return []
+
+  # Disable system C and C++ include directories, and add our own (using
+  # -idirafter so they are last, like system dirs, which allows projects to
+  # override them)
   C_INCLUDE_PATHS = [
```

Comment:
I'd rename these to be all lowercase (e.g. `c_include_paths`), because all caps says "global variable"

Reply:
Done